connectors/sources/dropbox.py

```diff
@@ -864,8 +866,14 @@ async def get_content(
         Returns:
             dictionary: Content document with _id, _timestamp and attachment content
         """
+        if not doit:
+            self._logger.debug(f"Skipping attachment downloading for {attachment['name']}")
+            return
+
         file_size = int(attachment["size"])
-        if not (doit and file_size > 0):
+
+        if file_size <= 0:
+            self._logger.warn(f"Skipping file '{attachment["name"]}' as file size is {file_size}")
```

Comment:
```suggestion
            self._logger.warning(f"Skipping file '{attachment["name"]}' as file size is {file_size}")
```

Use `warning` instead of `warn`